PRS9001-10000/PRS9199TaklaYo.xml

```diff
@@ -45,12 +46,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <body>
          <listPerson>
             <person sex="1">
-               <persName>
-                  
-                  Täklä Yoḥannǝs
-               </persName>
-               <occupation> mähari </occupation>
+               <persName xml:lang="gez" xml:id="n1">ተክለ፡ ዮሐንስ፡</persName>
+               <persName xml:lang="gez" corresp="#n1" type="normalized">Täklä Yoḥannǝs</persName>
+               <occupation type="ecclesiastic">mahari </occupation>
```

Comment:
not mahāri?

Reply:
The manuscript on folio 1va gives marāhi. Is it a mistake by the scribe?

<img width="382" alt="Screenshot 2024_marahi Takla Yohannes" src="https://github.com/BetaMasaheft/Persons/assets/40291787/1c7eaaec-0b2c-4b2c-ae4d-8320a61deeb5">

Reply:
it is a different word, both exist. then write marāhi

Reply:
Yes, marāhi / marāḥi = 'leader' should be a possible term in that place.